### PR TITLE
New Entry Swift Set.intersection()

### PR DESCRIPTION
**Pull request by hcz**
* Create intersection.md

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swift/concepts/sets/terms/intersection/intersection.md
Comment (SSwiniarski) [inline content/swift/concepts/sets/terms/intersection/intersection.md:3]:
```suggestion
Description: 'Returns a new set of elements with the overlapping elements of two sets.'
```

File: content/swift/concepts/sets/terms/intersection/intersection.md
Comment (SSwiniarski) [inline content/swift/concepts/sets/terms/intersection/intersection.md:15]:
```suggestion
The **`.intersection()`** method returns a new set containing all elements of a set that also belong to the given sequence.
```

File: content/swift/concepts/sets/terms/intersection/intersection.md
Comment (SSwiniarski) [inline content/swift/concepts/sets/terms/intersection/intersection.md:20]:
```suggestion
newSet = mySet.intersection(mySequece: Sequence)
```

File: content/swift/concepts/sets/terms/intersection/intersection.md
Comment (SSwiniarski) [inline content/swift/concepts/sets/terms/intersection/intersection.md:46]:
```suggestion
> **Note:** A set is an unordered data structure, so output elements may be ordered differently.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@hcz I completed a first review and left a few comments.

**Comment by hcz** [comment]
Fixed all 08e84066

**Review by SSwiniarski — APPROVED** [review]
@hcz, Looks good for a second review.

File: content/swift/concepts/sets/sets.md
Comment (yangc95) [inline content/swift/concepts/sets/sets.md:85]: Could we actually remove this table? I created issues for the additional methods listed in this table.

Instead, let's keep the header and introduce the methods with a sentence, followed by `:` a colon to lead into the table that will populate as per the terms folder.

File: content/swift/concepts/sets/terms/intersection/intersection.md
Comment (yangc95) [inline content/swift/concepts/sets/terms/intersection/intersection.md:23]:
```suggestion
Sets `mySet` and sequence `mySequence` must have countable number of members.
```

**Review by yangc95 — COMMENTED** [review]
@hcz thank your for this well-written PR! I had minor comments before merging

File: content/swift/concepts/sets/sets.md
Comment (hcz) [reply content/swift/concepts/sets/sets.md:85]: To be clear, should it be like this, similar to the arrays?

> ### Methods
> 
> There are many set manipulation methods available in the Swift Standard Library, including generic Collection based methods. Some of these include testing the contents of sets (e.g. `contains()` and `.isEmpty`) while others can modify or manipulate the set entirely (e.g., `map()`, `reduce()`, and `sort()`). There is a list of methods applicable to the pair of sets (e.g., `.intersection()` or `.subtracting()`)
> 
> Below are some methods available for sets:

File: content/swift/concepts/sets/sets.md
Comment (yangc95) [inline content/swift/concepts/sets/sets.md:79]:
```suggestion
There are many set manipulation methods available in the Swift Standard Library, including generic `Collection`-based methods. Some of these include testing the contents of sets (e.g. `.contains()` and `.isEmpty`) while others can modify or manipulate the set entirely (e.g., `.map()`, `.reduce()`, and `.sorted()`). There is also a list of methods applicable to a pair of sets (e.g., `.intersection()` or `.subtracting()`).
```

**Review by yangc95 — APPROVED** [review]
Looks good! I made some changes here: [9fe413f](https://github.com/Codecademy/docs/pull/1118/commits/9fe413f68cd854b206a450fee0d264d6370452c8) & [e13a3a9](https://github.com/Codecademy/docs/pull/1118/commits/e13a3a952bdf1dc43db4e57196f149d772101ec8)

**Comment by yangc95** [comment]
@hcz 🥳 live link: https://www.codecademy.com/resources/docs/swift/sets/intersection

cc: @SSwiniarski

**Pull request merged**
